tests/lua/spectre_common_test.lua

```diff
@@ -361,19 +360,17 @@ describe("spectre_common", function()
             end)
             it("Errors out when multiple destination values are provided", function()
                 local should_proxy, err = spectre_common.is_request_for_proxied_service('GET', {
-                    ['X-Smartstack-Source'] = 'src',
                     ['X-Smartstack-Destination'] = {'dst1', 'dst2'},
                 })
                 assert.are.equal(false, should_proxy)
                 assert.are.equal('X-Smartstack-Destination has multiple values: dst1 dst2;', err)
             end)
             it("Combines error messages when multiple sources AND destinations are provided", function()
```

Comment:
I'd just remove this test since it now does the exact same thing than the one above it.